boards/cc1352p-launchpad/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+  USEMODULE += netif
+  USEMODULE += cc26x2_cc13x2_rf
```

Comment:
You should only need this line, the other two should be resolved by `Makefile.dep`

Reply:
Thanks for pointing it up! I've fixed it